core/src/main/scala/org/broadinstitute/dsde/rawls/policy/PolicyService.scala

```diff
@@ -63,4 +65,15 @@ class PolicyService(tpsDAO: TpsDAO)(implicit val ec: ExecutionContext) {
 
   def getPao(objectId: UUID, ctx: RawlsRequestContext): Future[Option[TpsPaoGetResult]] =
     tpsDAO.getPao(objectId, ctx)
+
+  def deleteWorkspacePao(workspaceId: UUID, ctx: RawlsRequestContext): Future[Unit] =
+    Future {
+      try
+        tpsDAO.deletePao(workspaceId, ctx)
+      catch {
+        case ex: ApiException =>
+          logger.error(s"ApiException occurred while deleting PAO: ${ex.getMessage}", ex)
+        // Do not rethrow the exception, effectively ignoring it
+      }
+    }
```

Comment:
You'll need to change the error handling here. `tpsDAO.deletePao` returns a `Future`, so you won't be able to catch it with a try-catch like you have here. Instead, I'd recommend checking out `recover`, as it will be a failed `Future` containing the exception if the workspace PAO doesn't exist.

You'll also need to remove the `Future` wrapping the error-handling. `Future[Future[A]]` is a type that you should avoid as it won't run predictably